plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -626,17 +638,18 @@ private void addSplitPluginDependencies(String thisPlugin, MavenRunner.Config mc
                             continue; // already handled
                         }
                         // We ignore the declared dependency version and go with the bundled version:
-                        Plugin depBundledP = otherPlugins.get(depPlugin);
-                        if (depBundledP != null) {
-                            System.out.println("Adding " + depPlugin + " since it was a dependency of " + plugin);
-                            toAdd.put(depPlugin, new VersionNumber(depBundledP.version));
-                        }
+                            Plugin depBundledP = otherPlugins.get(depPlugin);
+                            if (depBundledP != null) {
+                                System.out.println("Adding " + depPlugin + " since it was a dependency of " + plugin);
+                                toAdd.put(depPlugin, new VersionNumber(depBundledP.version));
+
+                            }
```

Comment:
Nope, looks like somehow a whitespace change slipped in *sigh.*  Pushing a commit to fix it.